en/flying/package_delivery_mission.md

```diff
@@ -0,0 +1,84 @@
+# Package delivery mission planning
+
+A package delivery mission allows user to plan & execute a package delivery. Note that this is only supported for vehicles that has a [delivery mechanism hardware](../peripherals/gripper.md) configured & installed.
+
+
+## Operator control configuration
+
+Before using the package delivery feature, if feasible please setup the [gripper action button](../peripherals/gripper.md#griper-action-joystick-button-mapping-in-qgc) to have a manual control over the Gripper's state.
+
+
+## Delivery Mission Planning and Configuration (Gripper)
+
+Currently the package delivery feature only supports a [Gripper mechanism](../peripherals/gripper.md). Therefore the following documentation focuses on configuring a delivery mission using a Gripper.
+
+The package delivery feature using a gripper is embedded in the action waypoint "Gripper Mechanism". To enable different cases for different vehicle types and delivery behaviors, it has been decoupled as a separate waypoint. This is illustrated in the diagram below, where a clear **pre-waypoint** and **action-waypoint** cascade structure allows enabling different delivery use cases.
+
+![Package delivery action waypoint item methodology](../../assets/flying/package_delivery_action_waypoint_item.jpg)
```

Comment:
I understand the point of this diagram - the actual release is preceded by some other operation. The diagram shows an MC doing land, and a VTOL doing land or drop. But an MC is a "VTOL" so an MC can do land and drop too.

Reply:
Valid point, it can also drop from the sky. I will update the diagram!

Reply:
Cool. Do we also want to mention winches here?

Reply:
@junwoo091400 Do we also want to mention winches here?